tests/test_speed.py

```diff
@@ -27,6 +27,16 @@ def test_SpeedAndBufferSet(self, dvs, testlog):
         asic_port_table = swsscommon.Table(adb, "ASIC_STATE:SAI_OBJECT_TYPE_PORT")
         asic_profile_table = swsscommon.Table(adb, "ASIC_STATE:SAI_OBJECT_TYPE_BUFFER_PROFILE")
 
+        asic_port_records = asic_port_table.getKeys()
+        for k in asic_port_records:
+            (status, fvs) = asic_port_table.get(k)
+            assert status == True
+            for fv in fvs:
+                if fv[0] == "SAI_PORT_ATTR_SPEED":
+                    configured_speed_list.append(fv[1])
```

Comment:
very minor one, "break" here after this line to save some cycles.  
Add a brief comments for this block that we use one of the port speed as the default speed

Reply:
Thought so, max number of fields is NOT even 10, so its a constant time, its a minuscule gain! hence felt not to add an extra line of code! Sure let me add comments